frontend/src/Frontend/UI/Transfer.hs

```diff
@@ -660,7 +659,7 @@ transferDialog model netInfo ti ty fks tks _ = do
         let nextScreen = ffor (tag allDyns next) $ \case
               (_,Nothing,_,_) -> Workflow $ pure (mempty, never)
               (p,Just sc,gp,ss) -> previewDialog model netInfo ti payload sc $
-                                     crossChainTransferAndStatus model netInfo ti sc gp ss
+                                     crossChainTransferAndStatus model netInfo ti sc gp ss meta
```

Comment:
Instead of doing `sample` inside the `crossChainTransferAndStatus` function I think we should add those dyns to `allDyns` a few lines up because that's the place where we're getting our event that determines when all these values should be sampled.

Reply:
Good catch. Done